core/services/ocr2/delegate.go

```diff
@@ -1135,14 +1156,33 @@ func (d *Delegate) newServicesOCR2Keepers20(
 		d.cfg.JobPipeline().MaxSuccessfulRuns(),
 	)
 
-	return []job.ServiceCtx{
+	automationServices := []job.ServiceCtx{
 		job.NewServiceAdapter(runr),
 		runResultSaver,
 		keeperProvider,
 		rgstry,
 		logProvider,
 		pluginService,
-	}, nil
+	}
+
+	//var services evm.AutomationServices
+	if cfg.CaptureAutomationCustomTelemetry {
```

Comment:
can we remove from 2.0, not much point of adding there we want to deprecate 2.0 soon